.github/workflows/aws_tfhe_integer_tests.yml

```diff
@@ -4,6 +4,7 @@ env:
   CARGO_TERM_COLOR: always
   ACTION_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
   RUSTFLAGS: "-C target-cpu=native"
+  REPOSITORY: ${{ inputs.fork_user || github.repository_owner}}/tfhe-rs
```

Comment:
not compatible with internal repo, not sure how big of a problem it is

Reply:
I think the best and cleanest way to deal with it is to pass `fork_repo` instead of `fork_user`. It would imply to modify the code on Slab side a little bit, but it would avoid complicated way to deal with internal repo.

Reply:
up to you to chose the best solution